ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/MapContainerFragment.kt

```diff
@@ -164,6 +166,23 @@ class MapContainerFragment @Inject constructor(private var mapsRepository: MapsR
     }
     mapContainerViewModel.mbtilesFilePaths.observe(this) { map.addLocalTileOverlays(it) }
 
+    //    mapFragment.cameraMovedEvents
```

Comment:
Remove?

Reply:
This was already removed in later commits